### PR TITLE
Show commit list and commit diff 

### PR DESCRIPTION
**Pull request by rahulrrixe**
This pull request adds commit list view and commit diff view for the git repository.

### CODE REVIEW

File: fresque/gitview.py [hunk 1]
```diff
@@ -219,3 +220,121 @@ def view_raw_file(repo, identifier, filename):
...
     return (data, 200, headers)
+
+
+@APP.route('/git/<repo>/commits/<branchname>/')
+@APP.route('/git/<repo>/commits/<branchname>')
+def view_commit_list(repo, branchname=None):
+    """ Render a commit in a repo
+    """
+    """ Displays the commits of the specified repo.
+    """
```
Comment (abompard) [inline fresque/gitview.py]: Please merge the docstrings.

**Comment by rahulrrixe** [comment]
@abompard completed :+1:  Travis CI failing because of some problem in pygit2 and cffi installation.

**Comment by ralphbean** [comment]
@abompard, @pypingou, any further comment here?

**Comment by pypingou** [comment]
Arf this is conflicting now :(

@rahulrrixe any chances you could look at rebasing this PR?

**Comment by pypingou** [comment]
@cverna solved the conflicts while keeping the change in #18 

Let's close this then :)

**Pull request closed without merging**